bootstrap/apollo-env.sh

```diff
@@ -14,3 +17,23 @@ if [[ -z "${color_start-}" ]]; then
   export color_norm="${color_start}0m"
 fi
 
+# Change logging levels of called components at a global level
+# if unset allow for existing selective logging of components
+case "${APOLLO_LOG}" in
+  "")
+    # Do nothing in this instance
+  ;;
+  0)
+    # Force minimal logging
+    echo "Forcing reduction of component logging"
+    unset TF_LOG
```

Comment:
We still need to have a consitent behaviour here. So may be just set the variables to empty:
export ANSIBLE_LOG=
export TF_LOG=

Once we test it with this change lgtm.